make.js

```diff
@@ -40,8 +40,22 @@ target.build = function() {
     rm(path.join(buildPath, 'index.*'));
 }
 
-// test is just building samples
+
+target.units = function() {
+    pushd('test');
+    run('npm install ../_build');
+    popd();
+
+    console.log("-------Unit Tests-------");
+    run('tsc -p ./test/units');
+    run('mocha test/units');
+}
+
 target.test = function() {
+    target.units();
```

Comment:
In the previous version you had one more test here, did you mean to remove that too?

Reply:
Yeah, that was the test that had been a placeholder which I got rid of